apps/guardian-ui/src/components/SetConfiguration.tsx

```diff
@@ -215,6 +221,19 @@ export const SetConfiguration: React.FC<Props> = ({ next }: Props) => {
           />
           <FormHelperText>{t('set-config.admin-password-help')}</FormHelperText>
         </FormControl>
+        <FormControl>
+          <FormLabel>{t('set-config.confirm-password')}</FormLabel>
+          <Input
+            type='password'
+            value={confirmPassword}
+            onChange={(ev) => setConfirmPassword(ev.currentTarget.value)}
+          />
+          <FormHelperText>
+            {passwordsMatch && password.length > 0
+              ? 'Passwords match!'
+              : 'Passwords do not match!'}
```

Comment:
Text needs localization

Reply:
done, used google translate for korean do we need to check that with someone?

Reply:
Nope, the majority of it is machine-sourced right now so no sweat there, someone can PR better translations if they'd like.